lib/global_defs.rb

```diff
@@ -1593,7 +1593,7 @@ def respond(first = "", *messages)
     end
     messages.flatten.each { |message| str += sprintf("%s\r\n", message.to_s.chomp) }
     str.split(/\r?\n/).each { |line| Script.new_script_output(line); Buffer.update(line, Buffer::SCRIPT_OUTPUT) }
-    str.gsub!(/\r?\n/, "\r\n") if $frontend == 'genie'
+    # str.gsub!(/\r?\n/, "\r\n") if $frontend == 'genie'
     if $frontend == 'stormfront' || $frontend == 'genie'
```

Comment:
However if I comment out the check for `genie` as front end in this line, the double spacing is gone, but the nice horizontal formatting is then affected:

![image](https://user-images.githubusercontent.com/6467969/171859076-9c3d4f0d-c74f-4521-8860-388afd2c9168.png)

Reply:
The Genie team is looking at the mono font issue as a problem on our side. We would prefer to consume what's sent by Lich the same as any other FE does, rather than have exceptions coded into Lich.